x-pack/plugins/actions/docs/openapi/components/schemas/config_properties_webhook.yaml

```diff
@@ -1,5 +1,55 @@
 title: Connector request properties for a Webhook connector
 description: Defines properties for connectors when type is `.webhook`.
 type: object
-additionalProperties: true
-# TO-DO: Add the properties for this connector.
\ No newline at end of file
+properties:
+  authType:
+    type: string
+    enum:
+      - webhook-authentication-basic
+      - webhook-authentication-ssl
+      - "null"
+    description: >
+      The type of authentication to use: basic, SSL, or none.
+  ca:
+    type: string
+    # description: An encrypted version of the certificate authority file, which is used to sign and validate certificates.
```

Comment:
>     # description: An encrypted version of the certificate authority file, which is used to sign and validate certificates.

I've commented out the descriptions for now since I need clarification about how these strings are generated from the files that I uploaded in Kibana.

Reply:
Thanks for the clarification! Fixed in 40dc7ea337e34d50b067d70f0f8ff0fe09048ef5